source/shared/net/guides/become-baker.rst

```diff
@@ -341,6 +341,30 @@ first need to shut down the current running node. To do this, either press ``Ctr
 
 When you've placed the file in the appropriate directory, which is what you did you did in the previous command when you specified the output file, start the node again using ``concordium-node``.
 
+Configure a baker
```

Comment:
Note that add baker will require `--open-delegation-for all|none|existing`, `--baker-url URL`, `--delegation-transaction-fee-commission DECIMAL-FRACTION`, `--delegation-baking-commission DECIMAL-FRACTION` and `--delegation-finalization-commission DECIMAL-FRACTION` from after the protocol update.